npm/private/npm_translate_lock_generate.bzl

```diff
@@ -271,10 +271,7 @@ def npm_link_all_packages(name = "node_modules", imported_links = []):
                 ),
             )
 
-        stores_bzl.append("""        store_{i}(name = "{{}}/{pkg}".format(name))""".format(
```

Comment:
The store macro only used this `name` to extract the `/{pkg}` and determine the package name.